requirements/prod.txt

```diff
@@ -1,5 +1,6 @@
 -r compiled.txt
-amo-validator==1.10.15
+#amo-validator==1.10.15
+-e git+https://github.com/magopian/amo-validator.git@3250950f09b332d403ce7287ec5dd6ff9b7fe034#egg=amo-validator
```

Comment:
it's needed for travis to work correctly. It'll be removed before merging, see the PR's description